common.c

```diff
@@ -314,19 +319,13 @@ get_config_file()
 	rtr_getenv_t real_getenv;
 	int olderrno;
 
-	if (!get_tracing_enabled())
```

Comment:
Looks good then. I was trying to follow the call tree but got confused :)

I would certainly use assert, I had an assert in RETRACE_REPLACE at some point to check that we were actually finding the functions, but seems like it never made its way to master.